crowdin.yml

```diff
@@ -0,0 +1,3 @@
+files:
+  - source: pcsx2-qt/Translations/*.ts
+    translation: '/pcsx2-qt/Translations/PCSX2Emulator_%two_letters_code%.ts'
```

Comment:
Is it worth shortening PCSX2Emulator to something like "pcsx2lang" or "pcsx2"?

Reply:
I'd use `pcsx2-qt` to be consistent with the rest of the directory structure.